app/views/users/edit.html.haml

```diff
@@ -20,9 +20,10 @@
         = render :partial => 'admin_for_projects', :locals => { :f => form, :projects => @projects }
       - if current_visitor.has_role?("admin", "manager")
         = render :partial => 'researcher_for_projects', :locals => { :f => form, :projects => @projects }
+        = render :partial => 'project_cohorts', :locals => { :f => form, :projects => @projects }
       - elsif current_visitor.is_project_admin?
         = render :partial => 'researcher_for_projects', :locals => { :f => form, :projects => current_visitor.admin_for_projects }
+        = render :partial => 'project_cohorts', :locals => { :f => form, :projects => current_visitor.admin_for_projects }
```

Comment:
Can you use a policy_scope here for Projects? Then you could combine these two blocks.